smoke-tests-otel-starter/src/test/java/io/opentelemetry/smoketest/OtelSpringStarterSmokeTest.java

```diff
@@ -234,4 +255,56 @@ void shouldSendTelemetry() {
         .containsEntry(
             CodeIncubatingAttributes.CODE_NAMESPACE, "org.springframework.boot.StartupInfoLogger");
   }
+
+  @Test
+  @org.junit.jupiter.api.Order(2)
+  void restTemplateClient() {
+    reset(); // ignore the telemetry from application startup
```

Comment:
`reset()` is already called after each test (see `@AfterEach`). `reset()` call should only be required for the  `restTemplateClient()` test method.  Perhaps remove the `@AfterEach` method?

Reply:
great observation!